tests/functional/object/mcg/test_noobaa_endpoint_keda.py

```
@@ -0,0 +1,235 @@
+        # 2. Wait for scale-up
+        try:
+            for endpoint_pods in TimeoutSampler(
+                timeout=300,
+                sleep=30,
+                func=get_pods_having_label,
+                label=constants.NOOBAA_ENDPOINT_POD_LABEL,
+                namespace=namespace,
+            ):
+                count = len(endpoint_pods)
+                pod_names = ", ".join(p["metadata"]["name"] for p in endpoint_pods)
+                logger.info(f"Endpoint pods ({count}): {pod_names}")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)